modules/intentIqIdSystem.js

```diff
@@ -422,9 +431,19 @@ export const intentIqIdSubmodule = {
       }
     }
 
+    function updateGlobalObj () {
+      if (globalName) {
+        window[globalName].partnerData = partnerData
+        window[globalName].firstPartyData = firstPartyData
+        window[globalName].clientHints = clientHints
+        window[globalName].actualABGroup = actualABGroup
```

Comment:
Can you add checks?
`const partnerData = window[identityGlobalName]?.partnerData ??  {};`